include/SFML/Audio/SoundRecorder.hpp

```diff
@@ -134,7 +134,7 @@ class SFML_AUDIO_API SoundRecorder : AlResource
     /// \see getAvailableDevices, getDefaultDevice
     ///
     ////////////////////////////////////////////////////////////
-    bool setDevice(const std::string& name);
+    [[nodiscard]] [[nodiscard]] bool setDevice(const std::string& name);
```

Comment:
Is this one of those places in C++ where repeating something is necessary? 😁 I am surprised that there are no warning flags for duplicate attributes.

Reply:
Whoops... will fix